client/src/pages/ProfileSetting.js

```diff
@@ -0,0 +1,108 @@
+import React from "react";
+import Onboarding from "../components/Onboarding";
+import Grid from '@material-ui/core/Grid';
+import Typography from '@material-ui/core/Typography'; 
+import TextField from '@material-ui/core/TextField';
+import InputAdornment from '@material-ui/core/InputAdornment';
+import DropdownSelect from '../components/DropdownSelect';
+
+const moment = require('moment-timezone');
+// const abbrs = moment.tz.zonesForCountry('CA').map((z) => moment.tz(z).zoneAbbr());
+
+// Selected timezones only for Canada, otherwise there would be too many
+const timezones = moment.tz.zonesForCountry('CA').map((z) => {
+    return {
+        value:z,
+        label:`${z} (${moment(moment()).tz(z).format("HH:mm")})`
+    }
+});
+
+// const timezones = [
+//     {
+//       value: 'UTC',
+//       label: `UTC Time (${date.toLocaleTimeString('en-US', { hour12: false, 
+//         hour: "numeric", 
+//         minute: "numeric",
+//         timeZone: "UTC"})})`,
+//     },
+
+//     {
+//         value: 'EST',
+//         label: `EST Time (${date.toLocaleTimeString('en-US', { hour12: false, 
+//             hour: "numeric", 
+//             minute: "numeric",
+//             timeZone: "EST"})})`,
+//     },
+
+//     {
+//         value: 'PST',
+//         label: `PST Time (${date.toLocaleTimeString('en-US', { hour12: false, 
+//             hour: "numeric", 
+//             minute: "numeric",
+//             timeZone: "PST"})})`,
+//     },
+
+//     {
+//         value: 'MST',
+//         label: `MST Time (${date.toLocaleTimeString('en-US', { hour12: false, 
+//             hour: "numeric", 
+//             minute: "numeric",
+//             timeZone: "MST"})})`,
+//     },
+
+//     {
+//         value: 'GMT',
+//         label: `GMT Time (${date.toLocaleTimeString('en-US', { hour12: false, 
+//             hour: "numeric", 
+//             minute: "numeric",
+//             timeZone: "GMT"})})`,
+//     },
+//   ];
+
+
+function ProfileSetting() {
+
+    const [timezone,setTimezone] = React.useState(moment.tz.guess())
+
+    const handleChange = (event)=>{
+        setTimezone(event.target.value)
+    }
+
+    const path = "/calendar-confirm"
+
+    return (
+        <Onboarding title="Welcome to CalendApp!" activeStep={1} path={path}>
+            <Grid container item xs={12} justify="flex-start" alignItems="center">
+                <Grid item xs={4}>
+                    <Typography variant="subtitle2">Create your CalendApp URL:</Typography>
+                </Grid>
+
+                <Grid item xs={8}>
+                    <TextField
+                        InputProps={{
+                            startAdornment: <InputAdornment position="start">calendapp.com/</InputAdornment>,
+                        }}
```

Comment:
If you want to just add an [adornment to a text-field](https://material-ui.com/components/text-fields/#input-adornments), you can do:

```javascript
<TextField
  startAdornment={
    <InputAdornment position="start">...</InputAdornment>
  }
  {//...}
  />
```